src/listenMqtt.js

```diff
@@ -408,6 +418,67 @@ function parseDelta(ctx, globalCallback, defaultFuncs, v) {
   }
 }
 
+function markDelivery(ctx, defaultFuncs, threadID, messageID) {
```

Comment:
Why doesn't this use api.markAsDelivered?

Reply:
I could add a require statement with the markAsDelivered and markAsRead.

Reply:
It should be on the api object, right?

Reply:
There is no api object inside the api functions